pkg/cloudprovider/doc.go

```diff
@@ -15,4 +15,5 @@ limitations under the License.
 */
 
 // Package cloudprovider supplies interfaces and implementations for cloud service providers.
-package cloudprovider // import "k8s.io/kubernetes/pkg/cloudprovider"
+// To use: import cloudprovider "k8s.io/cloud-provider"
```

Comment:
Too late to change to `k8s.io/cloudprovider` ?

Reply:
Its not too late yet. I agree its awkward but it is also a common pattern we have across all of our staging repos. The problem is exacerbated because its a shallow repo. I had deliberately kept the common pattern but if we think is not a pattern we want to keep I can change it.

Reply:
I think it'd be better to change to one of:
* k8s.io/cloudprovider
* k8s.io/cloud-provider/cloudprovider

But I'm not 100% sure it is worth the effort if this is ready to go.